PocketKit/Sources/PocketKit/ReaderSettings/ReaderSettingsView.swift

```diff
@@ -5,44 +5,101 @@
-    }
-
     @Environment(\.presentationMode)
     private var presentationMode
-
+    @State private var dismissReason: DismissReason = .swipe
     @ObservedObject private var settings: ReaderSettings
 
     init(settings: ReaderSettings) {
         self.settings = settings
     }
 
     var body: some View {
-        NavigationView {
-            Form {
-                Section(header: Text(Localization.displaySettings)) {
-                    Picker(Localization.font, selection: settings.$fontFamily) {
-                        ForEach(Constants.allowedFontFamilies, id: \.name) { family in
-                            Text(family.name)
-                                .tag(family)
-                        }.navigationBarTitleDisplayMode(.inline)
+        Form {
+            Section(header: Text(Localization.Reader.Settings.title)) {
+                Picker(Localization.Reader.Settings.fontLabel, selection: settings.$fontFamily) {
+                    ForEach(settings.fontSet, id: \.rawValue) { family in
+                        Text(family.rawValue)
+                            .tag(family)
                     }
-
+                    .navigationBarTitleDisplayMode(.inline)
+                }
+                Stepper(
+                    Localization.Reader.Settings.fontSizeLabel,
+                    value: settings.$fontSizeAdjustment,
+                    in: settings.fontSizeAdjustmentRange,
+                    step: settings.fontSizeAdjustmentStep
+                )
+                .accessibilityIdentifier("reader-settings-font-size-stepper")
+                if settings.isPremium {
+                    Stepper(
+                        Localization.Reader.Settings.lineHeightLabel,
+                        value: settings.$lineHeightScaleFactorIndex,
+                        in: settings.settingIndexRange,
+                        step: settings.settingIndexStep
+                    )
+                    .accessibilityIdentifier("reader-settings-line-height-stepper")
                     Stepper(
-                        Localization.fontSize,
-                        value: settings.$fontSizeAdjustment,
-                        in: Constants.allowedAdjustments,
-                        step: Constants.adjustmentStep
+                        Localization.Reader.Settings.marginsLabel,
+                        value: settings.$marginsIndex,
+                        in: settings.settingIndexRange,
+                        step: settings.settingIndexStep
                     )
-                    .accessibilityIdentifier("reader-settings-stepper")
+                    .accessibilityIdentifier("reader-settings-margins-stepper")
+                    Picker(Localization.Reader.Settings.alignmentLabel, selection: settings.$alignmentOption) {
+                        ForEach(ReaderSettings.AlignmentOption.allCases, id: \.rawValue) { alignment in
+                            Text(alignment.rawValue)
```

Comment:
we might want to localize these alignment names if we decide to keep the feature

Reply:
Sounds like it's a keep. Go ahead and add them!